src/wallet/KintoWalletFactory.sol

```diff
@@ -9,9 +9,14 @@ import {UpgradeableBeacon} from "@openzeppelin/contracts/proxy/beacon/Upgradeabl
 import {SafeBeaconProxy} from "../proxy/SafeBeaconProxy.sol";
 
 import "../interfaces/IKintoID.sol";
+import "../interfaces/IFaucet.sol";
 import "../interfaces/IKintoWalletFactory.sol";
 import "../interfaces/IKintoWallet.sol";
 
+interface KintoIDWithAccessControl is IKintoID {
```

Comment:
not bad, but you could be just importing the interface from OZ:
```
import "@openzeppelin/contracts/access/IAccessControl.sol
```

Reply:
changing it